src/PrestaShopBundle/Resources/config/services/core/feature_flag.yml

```diff
@@ -2,6 +2,9 @@ services:
   _defaults:
     autowire: true
     autoconfigure: true
+  _instanceof:
```

Comment:
According to https://jolicode.com/blog/how-taggedlocator-can-help-you-design-better-symfony-application

You might be able to use AutoconfigureTag to avoid writing config, wdyt?

Reply:
After testing, it can't work like excepted with AutoconfigTag 😅

Reply:
Ah that's weird, but ok